ReactViewControl/ReactViewRender.cs

```diff
@@ -532,7 +532,7 @@ internal partial class ReactViewRender : UserControl, IDisposable {
                             var response = customResourceRequestedHandlers.Select(h => h(resourceKey, options)).FirstOrDefault(r => r != null);
 
                             if (response != null) {
-                                var path = uri.AbsolutePath;
+                                var path = uri.AbsoluteUri;
```

Comment:
Whats the difference, the extension isn't part of the AbsolutePath?

Reply:
The AbsolutePath is always `\\`, regardless of the extension

Reply:
But I don't see what the problem was...

Reply:
The extension (below this line) was always empty. This was making svg files not being properly loaded.

![image](https://user-images.githubusercontent.com/13751333/67390290-48642b80-f594-11e9-9f9d-3205b8ccbfd6.png)

Reply:
e.g. for uri `{custom://resource//?tree.svg}`:

- Absolute path is '//'
- Absolute uri is 'custom://resource//?tree.svg'